README.md

```diff
@@ -24,3 +25,11 @@ answer, including your reasoning, to this markdown file.
 ## Bonus
 
 Implement and analyze breadth-first search.
+
+## Runtime:
+
+The worst case for this algorithm, it being a search algorithm, is that the item we are searching for is not in the graph. The main iterator from the pseudocode is "While unvisited nodes remain." So in the process of searching for this item that is not in the graph, we will look at every single vertice exactly one time. (Technically we might check them more than once but it will go no further than checking if it has been visited, a constant-time operation.) So our runtime complexity for the worst-case scenario comes out to $\Theta(|V|)$ where $|V|$ is the number of vertices.
```

Comment:
Does the complexity not depend on the number of edges at all?

Reply:
I don't believe it does because each edge is traversed at most once and if an edge between two visited vertices is traversed the recursion ends as soon as it is realized that that vertex has already been visited.

Reply:
Well, let me ask another way -- what would the runtime be in a graph with 10,000,000 vertices but no edges?

Reply:
ahh fair point it would be constant time essentially wouldn't it?

Reply:
Well but what other quantity does the runtime depend on?